final_task/pycalc/pycalc.py

```diff
@@ -0,0 +1,190 @@
+"""This is a main module with general functions
+
+:author: Fyodorova Elena
+"""
+from re import findall
+import pycalc.Data as Data
+import argparse
+import importlib
+
+
+def pycalc(string: str):
```

Comment:
`pycalc` function output might be `string`, `bool` or `float`. It's better to implement functions, that will return a single type of an object.